src/sample/App.java

```diff
@@ -0,0 +1,32 @@
+package json.json;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import com.fasterxml.jackson.core.JsonGenerationException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+
+public class App
+{

```

Comment:
please be consistent, in java it is common to put a bracket on the same line as class, method name etc.